docs/UsingConfigManager/UsingConfigManager.xml

```diff
@@ -1119,19 +1121,35 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
           <title>DropZone Server List</title>
 
           <para>This tab allows you to add any servers that you wish to
-          configure as drop zones. To add a server to the available drop
-          zones. <orderedlist>
+          configure as a part of this drop zone. To add a server to the
```

Comment:
I think he meant to end with a colon.
"To add a server to the current drop zone:" 
However, the doc doesn't clarify the difference between adding more servers to one drop zone component and adding additional drop zone components.